amy/settings.py

```diff
@@ -19,17 +19,16 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = json.loads(os.environ.get('DEBUG', 'true'))
-# For deployment in production: 
-# DEBUG=false SECRET_KEY="..." ./manage.py runserver ...
+# SECURITY WARNING: don't run with DEBUG turned on in production!
+DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))
```

Comment:
It's there to handle 'true' / 'false' strings, ie. convert them to True/False objects in Python.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        From: Greg WilsonSent: Sunday, 1 March 2015 14:24To: swcarpentry/amyReply To: swcarpentry/amyCc: Piotr BanaszkiewiczSubject: Re: [amy] Add "AMY_" prefix to env vars DEBUG, SECRET_KEY (#206)In amy/settings.py:
> @@ -19,17 +19,16 @@
>  # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
>  
>  
> -# SECURITY WARNING: don't run with debug turned on in production!
> -DEBUG = json.loads(os.environ.get('DEBUG', 'true'))
> -# For deployment in production: 
> -# DEBUG=false SECRET_KEY="..." ./manage.py runserver ...
> +# SECURITY WARNING: don't run with DEBUG turned on in production!
> +DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))

Why are we doing a json.loads here?

—Reply to this email directly or view it on GitHub.